arm-operationalinsights/2015-03-20/swagger/OperationalInsights.json

```diff
@@ -0,0 +1,1110 @@
+          "description": "The boolean that indicates whether or not the field is stored."
+        },
+        "facet": {
+          "type": "boolean",
+          "description": "The boolean that indicates whether or not the field is a facet."
+        },
+        "ownerType": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The array of workflows containing the field."
+        }
+      },
+      "required": [
+        "facet",
+        "stored",
+        "indexed"
+      ],
+      "description": "Value object for schema results."
+    },
+    "SearchGetSchemaResponse": {
+      "properties": {
+        "__metadata": {
+          "x-ms-client-name": "Metadata",
+          "$ref": "#/definitions/SearchMetadata",
+          "description": "The metadata from search results."
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SearchSchemaValue"
+          },
+          "description": "The array of result values."
+        }
+      },
+      "description": "The get schema operation response."
+    },
+    "SearchHighlight": {
+      "properties": {
+        "pre": {
+          "type": "string",
+          "description": "The string that is put before a matched result."
+        },
+        "post": {
+          "type": "string",
+          "description": "The string that is put after a matched result."
+        }
+      },
+      "description": "Highlight details."
+    },
+    "SearchParameters": {
+      "properties": {
+        "top": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The number to get from the top."
+        },
+        "highlight": {
+          "$ref": "#/definitions/SearchHighlight",
+          "description": "The highlight that looks for all occurences of a string."
+        },
+        "query": {
+          "type": "string",
+          "description": "The query to search."
+        },
+        "start": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The start date filter, so the only query results returned are after this date."
+        },
+        "end": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The end date filter, so the only query results returned are before this date."
+        }
+      },
+      "description": "Parameters specifying the search query and range."
+    },
+    "StorageAccount": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The Azure Resource Manager ID of the storage account resource."
+        },
+        "key": {
+          "type": "string",
+          "description": "The storage account key."
+        }
+      },
+      "description": "Describes a storage account connection."
+    },
+    "StorageInsightStatus": {
+      "properties": {
+        "state": {
```

Comment:
Looks like an enum candidate? This also looks like a `"readOnly": true` property to me. It wouldn't make sense to ask the state of StorageInsightStatus from the customer.